cypress/integration/sensors.test.ts

```diff
@@ -1,7 +1,158 @@
+    cy.get("[data-cy='marker-circle-cluster-5248378642492082-13387527465820312']").click();
+    cy.get("[data-cy='marker-circle-cluster-5248381255975295-13387227058410645']").click();
+    cy.get("[data-cy='marker-circle-cluster-5248390403154306-1338707685470581']").click();
+    
+    map.get("[data-cy='same-position-markers-group']").should("exist");
+  });
+  it("should scroll element into view when map bubble is hovered", () => {
+    cy.viewport("macbook-13");
+    cy.visit("/sensors");
+
+    cy.get("[data-cy='marker-map-loaded']", { timeout: 16000 }).should("exist");
+    cy.get("[data-cy^='marker-circle']").should("have.length", 6);
+
+    cy.get("[data-cy='sensors-list-item']").should("have.length", 30);
+
+    // Single dot
+    const fadilItem = cy.get("[data-cy='sensors-list-item']:nth-child(23)").first();
+    fadilItem.scrollIntoView();
+    fadilItem.trigger("mouseover");
+    cy.get("[data-cy^='marker-circle'].bg-green").should("exist");
+
+    cy.scrollTo(0, 0);
+    cy.wait(1000);
+    cy.get("[data-cy^='marker-circle'].bg-green").trigger("mouseover");
+    cy.wait(1000);
+    cy.get("[data-cy='sensors-list-item']:nth-child(23)").should("be.inViewport");
+  });
+  it("should zoom the map when using buttons", () => {
+    cy.viewport("macbook-13");
+    cy.visit("/sensors");
+
+    cy.get("[data-cy='marker-map-loaded']", { timeout: 16000 }).should("exist");
+    cy.get("[data-cy^='marker-circle']").should("have.length", 6);
+
+    cy.get("button.mapboxgl-ctrl-zoom-in").click();
+    cy.wait(1000);
+    cy.get("button.mapboxgl-ctrl-zoom-out").click();
+  });
+});
+describe("The Sensors page - Mobile", () => {
+  it("should allow to switch between list and map", () => {
+    cy.viewport("iphone-3");
+    cy.visit("/sensors");
+
+    cy.findByRole("heading", { name: /Alle Sensoren/i }).should("exist");
+
+    cy.get("[data-cy='sensors-list-item']").should("have.length", 30);
+    cy.findByRole("button", { name: /1/i }).should("exist");
+    
+    cy.get("ul.flex").findAllByRole("listitem").should("have.length", 30);
+
+    cy.findByRole("button", { name: /2/i }).should("exist").click();
+
+    cy.get("[data-cy='map-list-switch-map-link']").click({ waitForAnimations: false });
+
+    cy.get("[data-cy='marker-map-loaded']", { timeout: 16000 }).should("exist");
+  });
+  it("should be able to search for a place", () => {
+    cy.viewport("iphone-3");
+    cy.visit("/sensors");
+
+    cy.get("[data-cy='map-list-switch-map-link']").click();
+
+    cy.get("[data-cy='marker-map-loaded']", { timeout: 16000 }).should("exist");
+
+    cy.findByRole("textbox").type("Platz d. Luftbrücke 4, 12101 Berlin, Germany");
+
+    cy.get("[data-cy='map-search-result']")
+      .first()
+      .click({ waitForAnimations: false });
+    
+    cy.get("[data-cy^='marker-circle']").should("have.length", 3);
+  });
+  it.only("should show a sensor thumbnail when marker is clicked on mobile", () => {
```

Comment:
Yes! This only happens on mobile! Regarding the test with the geo search field, there isn't any particular reason. There isn't neither any reason to run it on desktop only, and duplicating it feels wrong. Do you suggest we move it to desktop?

Reply:
Ah no, maybe it's a misunderstanding. I just noticed the `it.only` here which causes the other two tests to not run at all:

- "should allow to switch between list and map"
- "should be able to search for a place"

I was wondering if it is on purpose because e.g. they are currently not working?

Reply:
Oh! Yes! You're right. I removed the only keyword and now all tests are running